ports/raspberrypi/common-hal/microcontroller/Pin.c

```diff
@@ -63,12 +63,7 @@ void never_reset_pin_number(uint8_t pin_number) {
 }
 
 void reset_pin_number(uint8_t pin_number) {
-    if (pin_number >= 32
-#if TUD_OPT_RP2040_USB_DEVICE_ENUMERATION_FIX
-    // Pin 15 is used for Errata so we don't mess with it.
-        || pin_number == 15
-#endif
-        ) {
+    if (pin_number >= 32) {
```

Comment:
Should this be 30?

Reply:
And also above on line 58, should that be 30?

Reply:
Ya, though it isn't urgent. We don't have objects with numbers higher.